### PR TITLE
Link to property

### LINKED ISSUE #63: Linking to certain attributes
You did an cool job already via #56 which makes it already useful and nice for us and we will integrate it soon. I just had the idea that it would be useful to link directly to a property e.g. sometimes we get questions and we could then just say, please see the `allowed_vehicle` property and link it. Instead the customer now has to search manually through the docs to read everything and place it correctly.

### PR DESCRIPTION
**Pull request by fenyagg**
## What/Why/How?

Need the ability to make a reference to the property. 

Added links to properties. Added background highlighting of this property.
I added property names at the end of the path separeted with `|` (`%7C` after url encode). 
Discriminator and oneOf value separeted with `#`.
By default, all categories of this property are open.
If the reference to the property is in the response, the response will also be opened.

## Reference
closes #63

## Screenshots (optional)
add027d45a

## Check yourself

- [ ] Code is linted
- [ ] Tested
- [ ] All new/updated code is covered with tests

### CODE REVIEW

File: src/utils/helpers.ts [hunk 1]
```diff
@@ -217,3 +217,6 @@ export function isArray(value: unknown): value is any[] {
 export function isBoolean(value: unknown): value is boolean {
   return typeof value === 'boolean';
 }
+
+// get location hash without #
+export const getLocationHash = () => decodeURI(location.hash).substr(1);
```
Comment (fenyagg) [inline src/utils/helpers.ts:222]: There may be problems with server side rendering here (`location`). I think it needs to be wrapped in a try catch.

**Comment by fenyagg** [comment]
@AlexVarchuk @RomanHotsiy do you have any comments ?

**Comment by adamaltman** [comment]
Hi @fenyagg thank you for your contribution. However, we don't review/accept contributions for the features we highlight in our premium offerings: https://github.com/Redocly/redoc#redoc-vs-reference-vs-portals

We will make this policy more clear. Sorry about that.

**Comment by AlexVarchuk** [comment]
Hi @fenyagg. Thank you for your contribution.
Sorry again.

**Pull request closed without merging**